website/docs/Queries.md

```diff
@@ -193,6 +193,16 @@ const element = screen.getByRole('button');
 
 `name`: Finds an element with given `accessibilityRole` and an accessible name (equivalent to `byText` or `byLabelText` query).
 
+`disabled`: You can filter elements by their disabled state. The possible values are `true` or `false`. See [React Native's accessibilityState](https://reactnative.dev/docs/accessibility#accessibilitystate) docs to learn more about the `disabled` state.
```

Comment:
We should mention next to each item whether missing value is or isn't treated as false.